README.md

```diff
@@ -407,3 +407,10 @@ See the following for more:
 * You don't need to worry about setting up the gems, the container comes with them pre-installed
 
 [Back to top](#table-of-contents)
+
+# SMS Gateway for the immunization app
+  * You need to ask any of the team members for the api key
+  * To get the api key from https://gateway.seven.io
```

Comment:
This not clear please explain in detail how one can get the api key and also the url for the gateway

Reply:
I have revised the guide for getting the sms tell me if it makes sense